Sources/JXKit/JXConvertible.swift

```diff
@@ -43,18 +32,6 @@ extension Encodable where Self: JXConvertible {
     }
 }
 
-// We intentionally do not implement Equatable and Hashable to avoid confusion around the semantics of equatability.
```

Comment:
Moved this comment to JXValue because it didn't seem specific to the Convertible stuff.